proxy/envoy/testdata/egress-cb-policy.yaml.golden

```diff
@@ -0,0 +1,22 @@
+type: destination-policy
```

Comment:
We need CDS output corresponding to clusters in RDS (especially for auth and mixer).

Reply:
Yes I know. Don't know why it's missing. I think I didn't add those test cases